public/ng-views/maps.html

```diff
@@ -3,15 +3,18 @@
     <i class="fa fa-stop fa-stack-2x"></i>
```

Comment:
Just adding this at the top...

![image](https://cloud.githubusercontent.com/assets/6053396/8777793/3560e754-2ec8-11e5-9262-dab4da5c66ba.png)

If you want there to (I assume) be some sort of flat base level, we can do that but the leaflet map should not exist below it since that is both unnecessary rendering and a source of centering issues. Also you are having click-through issues with the divs